src/web/src/app/space/add/component.html

```diff
@@ -0,0 +1,45 @@
+<h2 i18n="@@SPACE.ADD.JOIN">Присоединиться</h2>
+<form class="join-form">
+	<mat-form-field class="join-input">
+		<mat-label i18n="@@SPACE">Пространство</mat-label>
+		<input matInput
+			type="text"
+			[formControl]="spaceAutocomplete"
+			[matAutocomplete]="auto"
+			i18n-placeholder="@@SPACE.ADD.JOIN.INPUT_PLACEHOLDER"
+			placeholder="Введите название пространства"
+			(input)="onAutoInputChange($event.target.value)"
+		>
+		<mat-hint class="red" *ngIf="!selectedSpace && isSpaces" i18n="@@SPACE.ADD.JOINFORM.CHOOSESPACE">Выберите пространство в выпадающем списке</mat-hint>
+		<mat-hint class="red" *ngIf="!isSpaces && !firstGetList" i18n="@@SPACE.ADD.JOINFORM.SPACENOTFOUND">Простраства не найдены</mat-hint>
```

Comment:
"Простраства не найдены" нужно показывать только тогда когда мы действительно проверили и получили ответ что они не найдены. Сейчас эта подсказка показывается вообще всегда, в том числе когда пользователь набирает новый текст.
```suggestion
		<mat-hint class="red" *ngIf="!isSpaces && !firstGetList" i18n="@@SPACE.ADD.JOINFORM.SPACENOTFOUND">Пространства не найдены</mat-hint>
```

Reply:
Вроде, поправил.